config/jobs/kubernetes-sigs/cluster-api-provider-ibmcloud/cluster-api-provider-ibmcom-periodics-main.yaml

```diff
@@ -0,0 +1,32 @@
+periodics:
+- name: periodic-cluster-api-provider-ibmcloud-coverage
+  interval: 12h
+  decorate: true
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-ibmcloud
+      base_ref: main
+      path_alias: "sigs.k8s.io/cluster-api-provider-ibmcloud"
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20220614-cde7860a33-1.23
+      imagePullPolicy: Always
```

Comment:
This can be changed to not always pull and save bw

Reply:
or just remove that to use the default policy like for the other jobs to avoid the load on the system.

Reply:
Fixed.